system/lib/libc/extras.c

```diff
@@ -31,3 +31,15 @@ long* _get_timezone() {
 void __lock(void* ptr) {}
 void __unlock(void* ptr) {}
 
+/* References to these longjmp- and exceptions-supporting things are generated
```

Comment:
why move into this file?

These symbols seems to live more logically in compiler-rt, which is a library that hold internal symbols that the compiler can generate during codegee.

Reply:
The main reason is that compiler-rt doesn't have a multithreaded variation atm, and adding one just for this seemed silly to me. Given libc already had a kind of "misc" file, this seemed reasonable to me. But I don't feel strongly, what do you prefer?